### DISCUSSION TITLE
BUG with right click adding to collections with text entered in search bar

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#16952] [Q&A] [answered]
I was going through adding all my custom maps for the ruleset Yoso to a collection and in order to filter for these maps I type adofai since all maps for yoso have the dif name adofai converted and after adding 6ish maps the collections tab on right click wouldn't show any of my collections untill after scrolling up a fair ways and scrolling back down. then it would show my collections for 1 map then be bugged again. still doing more testing to find out the scope of the problem

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16953
